### DISCUSSION TITLE
On IntelliJ, Copilot shows solution but doesn't always offer auto complete feature

### DISCUSSION THREAD

**Discussion by github-christophe-oudar** [community/community#7148] [Copilot Conversations] [answered]
On IntelliJ, sometimes, it can provide a solution but it doesn't show it as a possible "tab-able" solution to auto complete the solution directly. It requires to open the Github Copilot panel to see them.

Here an example:

![image](https://user-images.githubusercontent.com/90898980/139877269-b2044afa-b5d0-4909-aa90-dbbb874607e3.png)

Here my config:
```
IntelliJ IDEA 2021.2.3 (Community Edition)
Build #IC-212.5457.46, built on October 12, 2021
Runtime version: 11.0.12+7-b1504.40 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 11.5.2
plugin version Github copilot: 1.0.2
```

I tried `Copilot : show completions` as well but it doesn't do anything. Wouldn't be something that should offer the solution? If I start to add `_` to write the offered solution, then it offers the auto complete. I think it would be straightforward to have `Copilot : show completions` command show the solutions. 
I didn't try it on other IDEs to compare.

**Comment by jansorg** [comment, marked as answer]
Thanks! We're tracking this internally to investigate.  Using another IDEs shouldn't make a difference here.
Can you tell which SQL dialect you were using?

**Reply by github-christophe-oudar** [reply]
Hello, in that case, it's [Jinja](https://jinja.palletsprojects.com/) code for [DBT](https://www.getdbt.com/).
I might try to reproduce it on other language, I'll keep you posted.
